cinema_service/settings.py

```diff
@@ -121,7 +121,7 @@
 
 LANGUAGE_CODE = "en-us"
 
-TIME_ZONE = "UTC"
+TIME_ZONE = "Europe/Kiev"
```

Comment:
The `TIME_ZONE` is set to `Europe/Kiev`. Ensure this is the desired timezone for your application. If the application is intended for a different region, consider updating this setting.